cfme/tests/infrastructure/test_infra_quota.py

```diff
@@ -129,7 +129,7 @@ def _provisioner(template, provisioning_data, delayed=None):
     return _provisioner
 
 
-@pytest.mark.meta(blockers=[1287102])
+@pytest.mark.uncollectif(lambda: version.current_version() >= '5.5')
 def test_group_quota_max_memory_check_by_tagging(
```

Comment:
@psav  In 5.4 the default quota set is group quota, so this test is for group quota.

But now, in 5.5  the default quota source is tenant, the code needs be be re-factored and new tenant test cases needs to be added.